content/basics/use-ably.textile

```diff
@@ -0,0 +1,123 @@
+---
+title: How to use Ably
+meta_description: "A brief introduction on the best options available for using Ably"
+---
+
+This page will briefly cover some of the key APIs, SDKs, and adapters that Ably provides.
+
+h2(#sdks). Ably SDKs with Realtime and REST Interfaces
+
+Ably supports both Realtime, allowing persistent connections with low latency publishing and subscribing, and REST, communicating using the HTTP protocol and is effectively stateless, interfaces. Ably APIs are consistent across every language offering pub/sub functionality, presence, token and basic authentication, symmetric encryption and access to channel, presence and metrics history.
+
+Ably provides "a number of SDKS":https://ably.com/download in the most popular languages.
+
+h3(#realtime-interface). Realtime Interface
+
+The Ably Realtime interface establishes and maintains a persistent connection to Ably enabling low latency broadcasting and receiving of messages and presence state. 
+
+CAVEAT-INPUT
+
+ Reasons to use the Realtime interface:
+
+ * You are developing a client that needs to subscribe to messages in real time
+ * Your application needs to register its presence on a channel, or listen for others becoming present in real time
+ * You want to maintain a persistent connection to Ably, attach to one or more channels, and publish and subscribe to messages
+ * You want to publish messages at a very high volume
+ * You need to publish messages with very low latency
+ * You're running into your hourly API request limits
+
+The Ably Realtime interface is available in the most popular languages with "a number of SDKS":https://ably.com/download available.
+
+h3(#rest-interface). REST Interface
+
+The Ably REST interface provides a way for a wide range of server and client devices to communicate with the Ably service over REST. The REST API does not provide a realtime long-lived connection to Ably, but in all other respects is a simple subset of the full realtime messaging API.
+
+The primary use-case for the REST API is for servers that are part of the back-end of an application such as a web application, that publish messages, issue access tokens (temporary and client-specific) for its various clients, obtain message and presence history and retrieve statistics.
+
+CAVEAT-INPUT
+
+ Reasons to use the REST library:
+
+ * Your application server primarily issues tokens for clients and/or publish messages on channels
+ * Your application is mostly stateless
+ * You prefer synchronous requests, note that not all REST libraries are synchronous, but where the platform offers a synchronous and asynchronous approach, the REST libraries are more often synchronous
+
+The Ably REST interface is available in the most popular languages with "a number of SDKS":https://ably.com/download available.
+
+h2(#rest-api). Raw REST API
+
+Ably also provides a raw REST API where you don’t need to install any SDKs to use. 
```

Comment:
I think we could explain a little more about this, and possibly link to the OAS3 specification that we have. We also have the recombined REST API page we could link to.